.vscode/launch.json

```diff
@@ -0,0 +1,2272 @@
+{
```

Comment:
This file accounts for about half the diff here. Probably shouldn't be committed to the repository?